pages/glossary/label-in-name-algorithm.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Label in Name Algorithm
+key: label-in-name-algorithm
+unambiguous: true
+objective: false
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+To check whether an [element][] has <dfn>its label contained in its name</dfn>, follow this algorithm:
+
+Let 'label' be the [visible inner text][] of the target element.  Let 'name' be the [accessible name][] of the target element.  Both 'label' and 'name' are strings.
+
+Sub-algorithm to tokenize a string:
+
+- Convert the string to lower case.
+- For each character that either a) represents non-text content, or b) isn't a letter or a digit: replace that character with a space character.
+    - For a) Judgment of "non-text" probably can't be fully automated.  For example: "X" for "close" probably can be automated, but presumably there are more cases than this.
+    - For b) Use the Unicode classes Letter, Mark, and "Number, Decimal Digit [Nd]". (This will exclude hyphens, punctuation, emoji, and more.)
+- Remove all characters that are within parentheses (AKA round brackets).
```

Comment:
That was educational.  I admit that this PR will not handle those cases well.  But at least it will handle them as "false negatives", which are inevitable in ACT rules.  I think that this concern - and all of your concerns here - are valuable, but I don't think that this PR can reasonably be expected to handle them all.  This PR "closes" 5 issues, explicitly "does not handle" 3 more, and (I think) doesn't do any harm.  I'd like to improve the rule a little, then take it from there.  This PR has been open for 10 months.  "Perfect is the enemy of good" and all that.